environments/demo/service-core-compute-demo-internal.yml

```diff
@@ -14,5 +14,9 @@ vnet_links:
     vnet_id: "/subscriptions/b44eb479-9ae2-42e7-9c63-f3c599719b6f/resourceGroups/bastion-nonprod-rg/providers/Microsoft.Network/virtualNetworks/bastion-nonprod-vnet"
   - link_name: "pet-dev-vnet"
     vnet_id: "/subscriptions/58a2ce36-4e09-467b-8330-d164aa559c68/resourceGroups/pet_dev_network_resource_group/providers/Microsoft.Network/virtualNetworks/pet_dev_network"
-A: []
+A:
+  - name: camunda-api-demo
```

Comment:
Isn’t this done by private dns?

Reply:
Do u mean external dns ? Nah, we removed it for .internal as it was causing issues.

Reply:
ah k